src/App.tsx

```diff
@@ -1,30 +1,30 @@
+import {
+  NavLink,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import './App.scss';
-
-/*
-import { RouteComponentProps } from 'react-router-dom';
-
-type TabsPageProps = React.FC<RouteComponentProps<{ tabId: string }>>;
-const TabsPage: TabsPageProps = ({ match }) => {...};
-
-or
-
-import { useParams } from 'react-router-dom';
-
-const TabsPage = () => {
-  const { tabId } = useParams<{ tabId: string }>();
-  ...
-};
-*/
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import 'bulma';
+import { TabsPage } from './Components/TabsPage';
+import { NotFoundPage } from './Components/NotFounPage';
+import { HomePage } from './Components/HomePage';
 
 const App = () => (
   <div className="App">
-    <h1>Tabs with router</h1>
+    <header className="is-flex mb-4">
+      <NavLink to="/" className="navbar-item is-tab">Home</NavLink>
+      <NavLink to="/tabs" className="navbar-item is-tab">Tabs</NavLink>
```

Comment:
review the checklist, p.1

Reply:
Where can I find the checklist to this task? According to the task description everything is as it should be.